### DISCUSSION TITLE
Problems getting blank schema XML for customers in webservices

### DISCUSSION THREAD

**Discussion by DJ2412** [PrestaShop/PrestaShop#32866] [Q&A]
When I run http://localhost:80/api/customers?schema=blank I get an error.
He's the only one having problems. All others work fine even if I try to get it for a specific customer http://localhost:80/api/customers/2

<img width="292" alt="image" src="https://github.com/PrestaShop/PrestaShop/assets/39198090/ddd0f747-4e2b-4a78-8568-7fd8d4a47513">

When I try to create a client using the PSWebServiceLibrary I get the following when it tries to get api/customers?schema=blank

<img width="953" alt="image" src="https://github.com/PrestaShop/PrestaShop/assets/39198090/7dd11964-3b62-4b95-9870-de6a796eb6b4">

If I run it in Postman I get the following:

<img width="632" alt="image" src="https://github.com/PrestaShop/PrestaShop/assets/39198090/27fd98f1-0772-45cf-b44e-4b70e5ef5276">

<img width="631" alt="image" src="https://github.com/PrestaShop/PrestaShop/assets/39198090/21773b35-4dbd-4015-b9a9-6d9c76fda746">

Prestashop: 8.0.4
PHP: 8.1.20
MySQL version: 10.11.4-MariaDB

**Comment by matks** [comment]
Hello there a similar bug was posted here https://github.com/PrestaShop/PrestaShop/issues/32476

A Pull Request is being reviewed to fix it https://github.com/PrestaShop/PrestaShop/pull/32884

I close this discussion as there is already a bug report and a PR about it 😉
